content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -78,11 +77,15 @@ Notes:
 
 * **${CONFIG_URI}** - The GitHub address of the configuration YAML file that
   you want to use to deploy Kubeflow. The URI used in this guide is
-  {{% config-uri-existing-arrikto %}}.
+  {{% config-uri-istio-dex %}}.
   When you run `kfctl apply` or `kfctl build` (see the next step), kfctl creates
   a local version of the configuration YAML file which you can further
   customize if necessary.
 
+* **Disabling Istio Installation** - In the configuration file {{% config-file-istio-dex %}} you may choose to not install Istio by removing the applications `istio-crds` and `istio-install`, if your Kubernetes cluster has an existing Istio installation.
```

Comment:
s/Installation/installation

(The Kubeflow style guide recommends that we avoid capital letters unless we're referring to a product or brand name. Capital letters can be confusing if used for non-brand words.)